src/TerminalFighter/MenuState/MenuState.cpp

```diff
@@ -15,10 +15,22 @@ gamestates::GameStateName MenuState::run() {
     if (!settings_.load_str(SettingsSection::ASSET_PATHS, {"fonts", "default"}, default_font_path)) {
         LOG(FATAL) << "Failed to load default font in MenuState";
     }
+    int window_width, window_height;
+    if (!settings_.load_int(SettingsSection::VIDEO_SETTINGS, {"window", "width"}, window_width)) {
+        LOG(FATAL) << "Failed to load window width in MenuState";
+    }
+    if (!settings_.load_int(SettingsSection::VIDEO_SETTINGS, {"window", "height"}, window_height)) {
```

Comment:
We should make another namespace that is "ConfigPaths", so we get compiler errors if we type a path wrong, not a runtime error.